include/knowhere/config.h

```diff
@@ -501,6 +501,7 @@ const float defaultRangeFilter = 1.0f / 0.0;
 
 class BaseConfig : public Config {
  public:
+    CFG_INT dim;  // just used for config verify
```

Comment:
dim could be larger than the int32 max for sparse. should use a larger type

Reply:
currently sparse dim range is uint32max

Reply:
> currently sparse dim range is uint32max

Plz create an issue to track

Reply:
chatted offline, fixed in https://github.com/zilliztech/knowhere/pull/903